### PR TITLE
Alters JS script to add a random facts function and Changes CSS accordingly

### PR DESCRIPTION
**Pull request by SantiAce26**
Hi! So this just adds a random fact function to the JavaScript given to us and also changes CSS so that there's  a fact-container. There's also an image-thumbnail for the image in the index.html. Also I did not know that you had to confirm a pull request after you guys finished reviewing it, I assumed it did it by itself. So in conclusion I may have committed code for the index.html with the README file and had it merge before you guys reviewed, sorry!

### CODE REVIEW

**Review by kuanyiwu — APPROVED** [review]
LGTM, I love Demon Slayer too!

File: portfolio/src/main/webapp/style.css [hunk 1]
```diff
@@ -4,6 +4,25 @@
   width: 650px;
 }
 
+.image-thumbnails {
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: left;
+
```
Comment (frankyguerrero) [inline portfolio/src/main/webapp/style.css]: nit: delete line

File: portfolio/src/main/webapp/style.css [hunk 2]
```diff
@@ -4,6 +4,25 @@
...
+  max-height: 300px;
+  max-width: 500px;
+}
+
 #greeting-container {
   margin-top: 20px;
 }
+
+#fact-container
+{
```
Comment (frankyguerrero) [inline portfolio/src/main/webapp/style.css]: nit: put { on same line as #fact-container

File: portfolio/src/main/webapp/script.js [hunk 3]
```diff
@@ -26,3 +26,19 @@ function addRandomGreeting() {
   const greetingContainer = document.getElementById('greeting-container');
   greetingContainer.innerText = greeting;
 }
+
+function addRandomFact()
+{
```
Comment (frankyguerrero) [inline portfolio/src/main/webapp/script.js]: nit: move { to line 30

File: portfolio/src/main/webapp/script.js [hunk 4]
```diff
@@ -26,3 +26,19 @@ function addRandomGreeting() {
...
+
+    //List of facts
+    const facts =
+    ["I am Colombian B)", "I know Python,Java,C(Not an expert though)", "Demon Slayer is my favorite anime", "I love food...", "I like coding hehehe" ,
+    "I am double jointed in my fingers", "I'm highly interested in machine learning/deep learning", "Hey you landed on the special fact that you're cool"];
+
+    //Picks a random fact
+    const fact = facts[Math.floor(Math.random() *facts.length)];
```
Comment (frankyguerrero) [inline portfolio/src/main/webapp/script.js]: nit: add a space after *

File: portfolio/src/main/webapp/script.js [hunk 5]
```diff
@@ -26,3 +26,19 @@ function addRandomGreeting() {
...
+
+    // Add it to the page
+    const factContainter = document.getElementById('fact-container');
```
Comment (frankyguerrero) [inline portfolio/src/main/webapp/script.js]: nit: spelling factContainter -> factContainer (same below)

**Review by frankyguerrero — APPROVED** [review]
Left very small stylistic-related nits, but super excited about your website!!!

**Pull request merged**